luigi_pipeline/lib/model/seqr_mt_schema.py

```diff
@@ -348,9 +346,8 @@ def elasticsearch_row(ds):
             ds = ds.rows()
         # Converts nested structs into one field, e.g. {a: {b: 1}} => a.b: 1
         table = ds.drop('vep').flatten()
-        # When flattening, the table is unkeyed, which causes problems because our locus and alleles should not
+        # When flattening, the table is unkeyed, which causes problems because our row keys should not
         # be normal fields. We can also re-key, but I believe this is computational?
-        table = table.drop(table.locus, table.alleles)
-
-
-        return table
+        # PS: locus and alleles used to be the only possible row keys... however we're extending to drop any row key!
```

Comment:
I don't love comments that will live in perpetuity that say things like "this is what the behavior used to be". As a new user to a repo usually find it really confusing. Maybe a comment like "row key is often locus and allele, but does not have to be"

Reply:
👍